ui/src/semantic-ui/site/elements/label.overrides

```diff
@@ -1,3 +1,15 @@
 /*******************************
          Site Overrides
 *******************************/
+
+/* Frontsite */
+@{fs-parent-selector} {
+    .standard-number {
+        margin-left: 0;
+
+        .detail {
+            margin: -.5833em .5em -.5833em -.533em !important;
```

Comment:
Are these margins ok??? :) Looks like a weird number to me!

Reply:
it's because in the [semantic ui label](https://react.semantic-ui.com/elements/label/) the half label with the darker background is in the opposite side so I had to use the same margins with the label but in the opposite order.

Reply:
I see now, thank you for the explanation! Eventually I have other proposals then to avoid this:

![Screenshot 2021-03-17 at 11 43 30](https://user-images.githubusercontent.com/2089455/111455396-081bc500-8716-11eb-95d4-70563d933400.png)

```
<Label as='div' labelPosition='right'>
      <Label>
        Standard number
      </Label>
      <Label as='span' basic>
        ISO-21312-2131233-31212
      </Label>
    </Label>
```

Using a button but on hover it changes, maybe we can find a solution to that:

![Screenshot 2021-03-17 at 11 45 34](https://user-images.githubusercontent.com/2089455/111455657-5fba3080-8716-11eb-9486-d436913098a1.png)


```
<Button as='div' labelPosition='right'>
      <Button icon>
        Standard number
      </Button>
      <Label as='span' basic>
        ISO-21312-2131233-31212
      </Label>
    </Button>
```